.github/actions/download-vouch-extension-release/action.yaml

```
@@ -0,0 +1,42 @@
+name: Download Vouch Browser Extension latest release
+description: Download the latest release of the Vouch Browser Extension.
+inputs:
+  VOUCH_EXTENSION_DOWNLOAD_PAT:
+    required: true
+    description: 'Token to download the Browser Extension release from Vouch.'
+runs:
+  using: "composite"
+  steps:
+    - name: Create browser-extension directory
+      working-directory: ./packages
+      shell: bash
+      run: mkdir -p browser-extension
+    - name: Download Vouch Browser Extension latest release
+      working-directory: ./packages/browser-extension
+      shell: bash
+      env:
+        VOUCH_EXTENSION_DOWNLOAD_PAT: ${{ inputs.VOUCH_EXTENSION_DOWNLOAD_PAT }}
```

```suggestion
      env:
        VOUCH_EXTENSION_DOWNLOAD_PAT: ${{ secrets.VOUCH_EXTENSION_DOWNLOAD_PAT }}
```


We can use secret to simplify things

Another things is we shall use `browser-extension-nightly` (here, we still use `latest` what might be confusing), reflecting current progress on `main`. On the other hand, some of the test might use `browser-extension-latest` to check against recent extension deployed to chrome web store.

---

> We can use secret to simplify things

We cannot, `actions` do not have access to secrets, only workflows unfortunately.

---

Secrets can't be used in actions. That's why it has to be passed from workflow.
Currently we are using `nightly` everywhere.
Do we want to already add in this PR `latest` in some tests?

---

I think we should stick to only one channel (nightly) in this PR and leave latest for another PR.

---

Oh, you're right